client/js/utils.js

```diff
@@ -85,7 +85,7 @@ function updateTitle() {
 	let title = vueApp.appName;
 
 	if (vueApp.activeChannel) {
-		title = `${vueApp.activeChannel.channel.name} — ${vueApp.activeChannel.network.name} — ${title}`;
+		title = `${vueApp.activeChannel.channel.name} — ${title}`;
```

Comment:
@xPaw, do you know if we have access to the current window (not channel), at this point in the code?
I wanted to see if I could add special windows here, like `Settings - The Lounge` but I don't believe we have it yet, in which case it would have to wait for better window management.

Reply:
Windows are not ported yet, so no.